site/_includes/macros/navigation-tree-sections.njk

```diff
@@ -16,7 +16,7 @@ and marks it as active.
   {% if item.sections %}
     <button class="navigation-tree__link button" data-expandable aria-expanded="{{ item.active }}">
       <div class="navigation-tree__icon">
-        {{ icon('arrow-right', {label: 'toggle'}) }}
+        {{ icon('arrow-right', {label: 'toggle', hidden: true}) }}
```

Comment:
I wasn't quite sure why this change was added, but after [checking MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute) and viewing a page on the preview site it appears that this change hides non-meaningful content from the accessibility tree. Nice :)

Reply:
Yeah. I've removed the label too, since it's pointless if it's hidden.